src/foam/nanos/servlet/ImageServlet.java

```diff
@@ -69,6 +76,35 @@ protected void service(HttpServletRequest req, HttpServletResponse resp)
           IOUtils.copy(is, resp.getOutputStream());
           return;
         }
+      }else if ( reqPath.endsWith(".png") ){
```

Comment:
```suggestion
      } else if ( reqPath.endsWith(".png") ) {
```
Style guide should be on the foam repo (I think) or the nanopay getting started.